### PR TITLE
Published slides to Layer5 docs

### PR DESCRIPTION
**Pull request by nwanduka**
**Notes for Reviewers**

This PR fixes #

**[Signed commits](../blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by vishalvivekm** [comment]
@nwanduka 

Let's discuss it on the websites call.
Please add this as an agenda item in the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#heading=h.lohhtewfwima), if you would :)

File: content/en/cloud/security/images/Slide42.svg
[the remark this answers is not in the document]
Comment (nwanduka) [reply content/en/cloud/security/images/Slide42.svg:1]: Done

File: content/en/cloud/getting-started/getting-started-with-layer5-account.md [hunk 1]
```diff
@@ -33,6 +37,8 @@ Optionally, after you have configured 2FA, add a passkey to your account to enab
 To sign in to Layer5 Cloud, navigate to <https://cloud.layer5.io/login>. Use your new credentials to sign in. In case you have signed up with OAuth provider, you will be redirected to the provider's login page.
```
Comment (iArchitSharma) [inline content/en/cloud/getting-started/getting-started-with-layer5-account.md:37]: wrong url change it to` https://meshery.layer5.io/login`

File: content/en/cloud/getting-started/getting-started-with-layer5-account.md [hunk 2]
```diff
@@ -17,10 +17,14 @@ The first steps in starting with Layer5 are to create an account, verify your em
 To sign up for an account on Layer5 Cloud, navigate to <https://cloud.layer5.io/signup>.
```
Comment (iArchitSharma) [inline content/en/cloud/getting-started/getting-started-with-layer5-account.md:17]: change the url to - https://meshery.layer5.io/registration

File: content/en/cloud/getting-started/support.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+  ![Contact Us](/cloud/getting-started/images/Slide39.svg)
+
+- **Community Forum**: Join our vibrant [community forum](https://discuss.layer5.io/) where you can ask questions, share your experiences, and connect with other users. Our community members and moderators are here to help you and provide valuable insights and assistance.
+
+### Account Recovery
+
+If you've forgotten your Layer5 Cloud account password, don't worry. You can easily recover your account using the following steps:
+
+1. Go to the Layer5 Cloud [login page](https://cloud.layer5.io/login).
```
Comment (iArchitSharma) [inline content/en/cloud/getting-started/support.md:23]: change this too - https://meshery.layer5.io/login

**Review by iArchitSharma — CHANGES_REQUESTED** [review]

File: content/en/cloud/getting-started/getting-started-with-layer5-account.md (hunk 1 above)
Comment (nwanduka) [reply content/en/cloud/getting-started/getting-started-with-layer5-account.md]: Done for all

**Review by iArchitSharma — APPROVED** [review]
Thanks @nwanduka

**Pull request merged**